Corzbank.Web/ClientApp/src/app/header/header.component.html

```diff
@@ -116,10 +116,25 @@ <h3>Terminals</h3>
         <div class="exchange">
             <span>$ 32.487 / 36.579</span>
         </div>
-        <button class="account" (click)="openDialog()">
+        <div class="not-authenticated" (click)="openDialog()" *ngIf="!isAuthenticated">
             <i class="far fa-user"></i>
             <p>Sign in</p>
-        </button>
+        </div>
+        <div *ngIf="isAuthenticated">
```

Comment:
Instead of using opposite conditions, use else part inside the *ngIf, and ng-template with anchor for the other part

Usage: *ngIf="!isAuthenticated; else authenticatedUserTemplate"

<ng-template #authenticatedUserTemplate>
        .authentication-dropdown and so on
</ng-template>

Reply:
resolved